staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3545,11 +3545,11 @@ type TopologySpreadConstraint struct {
 	// "MaxSkew" on some topology.
 	// For example, in a 3-zone cluster, MaxSkew is set to 1, and pods with the same
 	// labelSelector spread as 3/1/1:
-	// +-------+-------+-------+
+	// |-------|-------|-------|
 	// | zone1 | zone2 | zone3 |
-	// +-------+-------+-------+
+	// |-------|-------|-------|
 	// | P P P |   P   |   P   |
-	// +-------+-------+-------+
+	// |-------|-------|-------|
```

Comment:
I didn't see anything related to changing `+` to `|` as part of your generator changes. Do you know why they are different here?

Reply:
Ah, I see in your description:
> I replaced + with | in text tables, because currently the generator is written to skip the line starting with +

Is `+` valid markdown? If so, I think it could be better to see it supported rather than avoided.

Reply:
https://michelf.ca/projects/php-markdown/extra/#table is an example of a Markdown flavor that has tables. `+` is not valid there.